parts/linux/cloud-init/nodecustomdata.yml

```diff
@@ -198,9 +196,7 @@ write_files:
   permissions: "0644"
   owner: root
   content: |
-    {{if HasDataDir -}}
-    root = "{{GetDataDir}}"
-    {{- end}}
+    {{if HasDataDir -}}root = "{{GetDataDir}}"{{end}}
```

Comment:
why is this different than L180

Reply:
trimming the entire line here is more difficult because there are no surrounding characters. suggestions welcome. 

if we trim the whitespace after the line, we end up with two merged lines. if we move it to the end of the file, we will add extra white space there. I was kind of surprised I couldn't find a simple way to do this right, maybe i'm just missing it?

Reply:
would add a test in RP to illustrate. It should be easy to fix similar to L180, you are right.

Reply:
https://diffy.org/diff/i3mp0rubyokiv3ny72p3hxgvi

should be better